screens/Groups.js

```diff
@@ -22,127 +84,129 @@ export default class GroupsScreen extends Component<Props> {
-            connect in friendships with others, and serve the world around you.
-          </Text>
-        </View>
+      <View style={styles.mainContainer}>
+        <ImageBackground
+          source={require('../assets/images/fall_leaves_bg.png')}
+          style={styles.backgroundImage}
+        />
+        <FlatList
+          keyExtractor={({ uuid }) => uuid}
+          data={this.state.groups}
+          ItemSeparatorComponent={() => <View style={styles.separator} />}
+          renderItem={({ item }) => {
+            const {
+              uuid,
+              groupname = '',
+              campus,
+              frequency,
+              interval,
+              daysOfWeek,
+              dayOfMonth,
+              meetingTime,
+              description,
+            } = item;
 
-        <View style={[styles.banner, styles.blue]}>
-          <Text style={[styles.heading, styles.center]}>Join a Group</Text>
-          <Text style={[styles.text, styles.center]}>
-            Ready to join a group? Explore groups by topic or location.
-          </Text>
-          <Button
-            title="Explore All Groups"
-            onPress={this.handleOpenGroups}
-            style={styles.button}
-          />
-        </View>
+            const groupTitleParts = groupname.split('-');
 
-        <View style={[styles.banner, styles.gray]}>
-          <Text style={[styles.text, styles.blackText]}>
-            Echo Groups are usually 8 to 15 people that meet regularly in homes,
-            parks, coffee shops, etc. around the Bay Area. Our groups range in
-            topics, with everything from hiking, to marriage focus, to financial
-            workshops, to in-depth Bible studies. Whether you are exploring
-            faith in God or you have been a long-time follower of Jesus, Echo
-            Groups are one of the best ways for you to grow in your journey of
-            faith, connect in friendships with others, and learn to make a
-            difference.
-          </Text>
-          <Text style={styles.semesterHeader}>Semester Cycles</Text>
-          <Text style={styles.semesterText}>
-            Echo Groups run in four-month cycles (semesters), which allows you
-            the option to switch groups according to your season of life.
-          </Text>
-          <View style={styles.season}>
-            <Text style={styles.seasonLabel}>Spring:</Text>
-            <Text style={styles.seasonText}>Feb – May (sign-ups in Jan)</Text>
-          </View>
-          <View style={styles.season}>
-            <Text style={styles.seasonLabel}>Summer:</Text>
-            <Text style={styles.seasonText}>Jun – Jul (sign-ups in May)</Text>
-          </View>
-          <View style={styles.season}>
-            <Text style={styles.seasonLabel}>Fall:</Text>
-            <Text style={styles.seasonText}>Sep – Dec (sign-ups in Aug)</Text>
-          </View>
-        </View>
-      </ScrollView>
+            groupTitleParts.shift();
```

Comment:
remove the group code thing from the group name title

we should probably do this on web too